scripts/install-lethekit

```diff
@@ -46,12 +46,33 @@ lkroot=${args[0]}
 aroot=${args[1]}
 libpath="${aroot}/libraries"
 
+# We need python to create libwally for arduino
+if ! python --version -v COMMAND &> /dev/null
```

Comment:
Should this be `if ! python3 --version -v COMMAND &> /dev/null`?

Reply:
tnx, I fixed the inconsistency below by calling python instead of python3. Both versions are ok.

If only python3 is installed on the system invoking `python` invokes `python3`